helper-cli/src/main/kotlin/commands/GetPackageLicensesCommand.kt

```diff
@@ -68,25 +65,15 @@ class GetPackageLicensesCommand : CliktCommand(
     ).convert { Identifier(it) }
         .required()
 
-    private val packageConfigurationOption by mutuallyExclusiveOptions(
-        option(
-            "--package-configuration-dir",
-            help = "The directory containing the package configuration files to read as input. It is searched " +
-                    "recursively."
-        ).convert { it.expandTilde() }
-            .file(mustExist = true, canBeFile = false, canBeDir = true, mustBeWritable = false, mustBeReadable = true)
-            .convert { PackageConfigurationOption.Dir(it) },
-        option(
-            "--package-configuration-file",
-            help = "The file containing the package configurations to read as input."
-        ).convert { it.expandTilde() }
-            .file(mustExist = true, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = true)
-            .convert { PackageConfigurationOption.File(it) }
-    ).single()
+    private val packageConfigurationDir by option(
+        "--package-configuration-dir",
+        help = "The directory containing the package configuration files to read as input. It is searched recursively."
+    ).file(mustExist = true, canBeFile = false, canBeDir = true, mustBeWritable = false, mustBeReadable = true)
+        .convert { it.expandTilde() }
 
     override fun run() {
         val scanResults = getStoredScanResults(packageId)
-        val packageConfigurationProvider = packageConfigurationOption.createProvider()
+        val packageConfigurationProvider = SimplePackageConfigurationProvider.forDirectory(packageConfigurationDir)
```

Comment:
My proposal is to do

    val packageConfigurationProvider = packageConfigurationDir?.let { DirectoryPackageConfigurationProvider(it) } ?: PackageConfigurationProvider.EMPTY

here and in the other places.